packages/@tinacms/graphql/src/primitives/resolver/index.ts

```diff
@@ -201,7 +201,6 @@ export class Resolver {
             await this.database.put(realPath, {
               _template: lastItem(template.namespace),
             })
-            return this.getDocument(realPath)
```

Comment:
And this was to fix the bug with `useDocumentCreator()` and `templates` instead of a `fields`.